WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -66,6 +67,7 @@ class SiteStatsInsightsTableViewController: UITableViewController {
         ImmuTable.registerRows(tableRowTypes(), tableView: tableView)
         loadInsightsFromUserDefaults()
         initViewModel()
+        tableView.estimatedRowHeight = 500
```

Comment:
🤔 Why 500 ;P? Is this needed for performance purposed or just any non-zero value to kick off some changes in how UITableView lays itself out?

Reply:
Oh sorry! I should have commented or something. That's related to [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/10649/commits/870d5a6029257e2eaa83b34ba70a85ad130d22d8) change. Basically, `FilterTabBar` was throwing constraint warnings without `estimatedRowHeight` set. The `TabbedTotalsCell` generally has a height of 400-ish (which thus far is the highest of all cells), so I set it at 500.

Reply:
👍